score-lang-cli/src/main/java/org/openscore/lang/cli/utils/CompilerHelperImpl.java

```diff
@@ -82,13 +83,24 @@ public SlangSource convert(File from) {
     }
 
 	@Override
-	public Map<String, ? extends Serializable> loadSystemProperties(List<String> systemPropertyFiles) {
+	public Map<String, ? extends Serializable> loadSystemProperties(List<String> systemPropertyFiles) throws IOException {
+		if(CollectionUtils.isEmpty(systemPropertyFiles)) {
+			Collection<File> spFiles = FileUtils.listFiles(new File("."), SP_EXT, true);
+			spFiles = select(spFiles, having(on(File.class).getPath(), containsString(SP_DIR)));
```

Comment:
again checking contatins meannins the user can't have the word properits in its path.
I find pretty limiting.